crates/volta-core/src/layout/unix.rs

```diff
@@ -0,0 +1,34 @@
+use std::path::PathBuf;
+
+use super::volta_home;
+use crate::error::ErrorDetails;
+use volta_fail::Fallible;
+
+pub(super) fn default_home_dir() -> Fallible<PathBuf> {
+    let mut home = dirs::home_dir().ok_or(ErrorDetails::NoHomeEnvironmentVar)?;
+    home.push(".volta");
+    Ok(home)
+}
+
+pub(super) fn default_install_dir() -> Fallible<PathBuf> {
+    // default location for the install directory
+    // (this will be the case for the majority of installs)
+    let home = volta_home()?.root();
+    if home.join("shim").exists() {
+        return Ok(home.to_owned());
+    }
+
+    // when an RPM is installed as root, the install_dir will be here for non-root users
+    // (this will be the case for some managed installs)
+    let rpm_home = PathBuf::from("/usr/bin/volta-lib");
```

Comment:
I don't understand this part... can you explain this a bit more? Is there something about RPMs described in the updates RFC?

It also seems good to have some documentation about managed installs in the web site, and this kind of thing seems like a good candidate for those docs. Not a blocker for this PR but we should make sure to work on it.

Reply:
We currently build an RPM package, using a script from #529 that puts the `volta` binary on `/usr/bin` and the shim binary in `/usr/bin/volta-lib`, so this detects that case. With the updates that will probably go away since we'll just put everything in one place and rename the shim to `volta-shim`.

I'll update the comment to mention that it's the current RPM installer and will likely change when the full managed installs behavior is implemented.